rpmlb/builder/base.py

```diff
@@ -16,29 +18,23 @@ class BaseBuilder:
     def __init__(self):
         pass
 
-    @classmethod
-    def get_instance(cls, name):
-        # TODO: Use reflection.
-        # class_name = 'rpmlb.builder.{0}.{1}Builder'.format(
-        #     name,
-        #     utils.camelize(name)
-        # )
-        # return utils.get_instance(class_name)
-        instance = None
-        if name == 'mock':
-            from rpmlb.builder.mock import MockBuilder
-            instance = MockBuilder()
-        elif name == 'copr':
-            from rpmlb.builder.copr import CoprBuilder
-            instance = CoprBuilder()
-        elif name == 'custom':
-            from rpmlb.builder.custom import CustomBuilder
-            instance = CustomBuilder()
-        elif name == 'dummy':
-            from rpmlb.builder.dummy import DummyBuilder
-            instance = DummyBuilder()
-        else:
-            raise ValueError('name is invalid.')
+    @staticmethod
+    def get_instance(name):
+        """Instantiate named builder.
+
+        Keyword arguments:
+            name (str): The name of the requested builder.
```

Comment:
If you already know this takes str, maybe use type annotation in function definition instead?